scalikejdbc-core/src/main/scala-3/scalikejdbc/DBConnectionMethods.scala

```diff
@@ -0,0 +1,16 @@
+package scalikejdbc
+
+trait DBConnectionMethods { self: DBConnection =>
+
+  def withReadOnly[A](execution: DBSession ?=> A): A =
```

Comment:
For the moment, I have add `with` prefix. but call for other opinions.

Reply:
How can this method be used?